getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1463,63 +1463,114 @@ freed.
 
 Signals
 ~~~~~~~
+It is often desired to send a notification from an instance to another
```

Comment:
That empty line below the title had a purpose :) Sphinx is very pedantic about style.